packages/dso-toolkit/src/components/card/card.scss

```diff
@@ -12,31 +12,29 @@
 }
 
 dso-card {
-  &[is-selectable],
-  &[has-image] {
+  &[is-selectable] {
```

Comment:
Eigenlijk moeten we deze styling ombouwen naar `::slotted` en `::part` selectors.

Laten we deze even achter de hand houden, we kunnen er ook een los issue van maken. Ik was uit het oog verloren hoeveel styling hier verkeerd zit.

Reply:
Zie https://github.com/dso-toolkit/dso-toolkit/issues/2828